readthedocs/templates/projects/project_import.html

```diff
@@ -70,7 +70,10 @@
       </span>
 
       <h1>{% trans "Import a Repository" %}</h1>
-
+      {% blocktrans %}
+          Here only public repositories are listed, If you want to find your private
+          repositories, you can find on <a href="https://travis-ci.com/">trvais-ci.com</a>
```

Comment:
The URL should point to [readthedocs.com](https://readthedocs.com/) the commercial service of ReadtheDocs. Also we need to make sure the message does not appear on the commercial website.

Reply:
Ohk sir , I will change the url from travis-ci.com to readthedocs.com and second thing , Can you please tell me that where can I access commercial website? in source code so that I can fix this.

Reply:
The source code from the commercial site is private, you can add an named block like https://github.com/rtfd/readthedocs.org/blob/c0bcb243b14b465695cacf9aab644da23c411ca7/readthedocs/templates/projects/project_import.html#L216 so we can override that value in the other template

Reply:
Ohk thanks @stsewd